pyproject.toml

```diff
@@ -5,6 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "changelist"
 version = "0.5rc0.dev0"
+# TODO add slots=True, kw_only=True to dataclasses starting with >=3.10
```

Comment:
Support for `slots` and `kw_only` in `dataclass` was only added in Python 3.10. Because changelist still supports 3.9 we can't add them yet.

However, I'd like to add have them eventually so I added a reminder. IMO, `kw_only` forces more expressive calls to init, and `slots` safes a bit of memory at no cost.